riscv-overlay-software-standard-draft.adoc

```diff
@@ -279,7 +279,7 @@ c.  RO Data – read-only data which is marked as overlay should be referenced w
 4.  We need to reserve N core registers. To be used only for the RT-Engine.
 Those registers will have a special purpose understood by the compiler, linker, debugger, and RT-Engine.
 Moreover, those registers form a _RT-Eng-Debugger-handshake_ between compiler, RT code, and debugger.
-Any library linked with an overlay enabled it to be compiled without using the reserved registers.
+Any library linked with an overlay enabled application compiled without using the reserved registers.
```

Comment:
`application compiled` -> `application must be compiled`

Reply:
I think we lost context here. The meaning of this sentence is to say that: if you have libs linked to your application, which use overlay, you must compile those libs again just without the "reserve regs"
I fixed it in: #9c35a854a4a8e789db3f6078b0bc88cc47605807